### PR TITLE
update attributes that send to scorecard milestone api

### PR DESCRIPTION
**Pull request by limkimsan**

### CODE REVIEW

File: app/db/schemas/schemaV5.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const schemaV5 = {
+  schema: schemaHelper.getSchemas(changedSchemas),
+  schemaVersion: 5,
+  migration: (oldRealm, newRealm) => {
+    if (oldRealm.schemaVersion < 5) {
+      const oldObjects = oldRealm.objects('Scorecard');
+      const newObjects = newRealm.objects('Scorecard');
+
+      for (let i = 0; i < oldObjects.length; i++) {
+        newObjects[i].uuid_on_app = !oldObjects[i].uuid_on_app ? uuidv4() : oldObjects[i].uuid_on_app;
```
Comment (kakada) [inline app/db/schemas/schemaV5.js:22]: @limkimsan what is the purpose of introducing uuid_on_app?

Comment (limkimsan) [reply app/db/schemas/schemaV5.js:22]: The uuid_on_app is for sending to scorecard milestone API, and it is used for defined different records of the scorecard in different devices.

Comment (limkimsan) [reply app/db/schemas/schemaV5.js:22]: Bong, do not merge this PR. I need to some changes on this PR.

Comment (limkimsan) [reply app/db/schemas/schemaV5.js:22]: Bong, I have finished updating the PR.

Comment (kakada) [reply app/db/schemas/schemaV5.js:22]: in term of sending uuid_on_app, please send the device_id instead if so

Comment (limkimsan) [reply app/db/schemas/schemaV5.js:22]: The current data send to milestone data includes both device_id and uuid_on_app. I will check with bong Sokly if we can use device_id only.

**Pull request merged**